app/Http/Controllers/AccountController.php

```diff
@@ -40,19 +38,13 @@ public function changePassword(Request $request)
         $user = User::find(Auth::user()->id);
 
         #update password through user model.
-        if (empty($request->inputPasswordOld1) === False) {
-            if ($request->inputPasswordOld1 != $request->inputPasswordOld2)
-                return redirect()->back()->with('danger', 'Old passwords do not match. Try again');
-            elseif (Hash::check($request->inputPasswordOld1, Auth::user()->password) === false)
-                return redirect()->back()->with('danger', 'Incorrect password! Try Again.');
```

Comment:
Could we put this check back? I just meant that we don't need to do the first if statement to prompt the user to fill out their old password twice on the page.